### ISSUE TITLE
Better design for Transaction History GUI

### ISSUE THREAD

**Issue by augustresende** [AugustoResende/RaiLightWallet#1]
Design concept:
![bpgsduafrd2mbm5yuyvjzg](https://user-images.githubusercontent.com/10731727/34682949-b87a3834-f487-11e7-83e8-f4f9adcf176f.png)

Currently preview:
![screenshot_45](https://user-images.githubusercontent.com/10731727/34798552-b23d57e0-f643-11e7-874c-8a402e3b2182.png)

**Comment by leonardobork** [comment]
What do you need?

**Comment by leonardobork** [comment]
CSS, HTML and JS only?

**Comment by augustresende** [comment]
Yes.

**Comment by leonardobork** [comment]
The grid? Have you an endpoint that returns your transactions?

**Comment by augustresende** [comment]
yes @leonardobork

**Comment by leonardobork** [comment]
yeah, i'll do it for you, i just gotta finish some work before that

**Comment by leonardobork** [comment]
do you want to add me as a collaborator or  should i just try a pr?

**Comment by augustresende** [comment]
PR first, 5 PR accepted = collaborator ;)

**Comment by leonardobork** [comment]
ok,i'll do it tonight

**Comment by leonardobork** [comment]
i can help you with server side stuff too, i strongly recommend you to divide code in small and concise archives and dirs, its better for readability and all that good stuff

**Comment by leonardobork** [comment]
if you want you can add me on skype and we can chat: leonardomerker@gmail.com

**Comment by AugustoResende** [comment]
i can't find you in skype..

**Comment by leonardobork** [comment]
gimme yours

**Comment by AugustoResende** [comment]
augusto.resende@outlook.com

**Comment by augustresende** [comment]
@leonardobork any progress?

**Comment by ant1m4tt3r** [comment]
Next week I'll work on this issue. Just familiarizing with electron first.

**Comment by jaonoctus** [comment]
@hugoazevedosoares it's not hard to do.
just run `npm install` or `yarn` first, then `./node_modules/electron/dist/electron .`

**Comment by ant1m4tt3r** [comment]
Thx, that I knew. I'm having trouble with debbuing it. What is the default password?

**Comment by augustresende** [comment]
Do not have a default password. You create when you open the program for the first time.

**Comment by augustresende** [comment]
If you have lost the password, just delete the data.db file. (But you will lost any funds, and the seed)

**Comment by ant1m4tt3r** [comment]
Working on. Can you send me the original layout files? I need the color scheme.

**Comment by ant1m4tt3r** [comment]
Using mocks now to develop and I've made some progress, but now I need some transactions to my wallet, so I can continue debbuging.

**Comment by ant1m4tt3r** [comment]
![wallet](https://user-images.githubusercontent.com/23644433/34956402-00e11876-fa10-11e7-91eb-63427a0906fa.png)

**Comment by augustresende** [comment]
Very good, can send me the CSS?

You are using ul and li?

**Comment by ant1m4tt3r** [comment]
No, still using a table

**Comment by ant1m4tt3r** [comment]
I can make a pull request, than you check if everything is ok with real transactions and not just mocks.

**Comment by augustresende** [comment]
Ok, make a pull request, i will change to ul and li, and test using real transactions ;)

**Comment by augustresende** [comment]
@hugoazevedosoares

**Comment by ant1m4tt3r** [comment]
Done! But I recommend not changing the table for a list, the table it's way more elegant and organized.

**Comment by ghost** [comment]
Hey, do you still need help with this? @AugustoResende @hugoazevedosoares

**Comment by NghiaSE** [comment]
@AugustoResende I think most people are familiar with Bitcoin most of them use Electrum wallet. Its GUI is elegant & easy to use.

![electrumltc1](https://user-images.githubusercontent.com/27041825/36930863-f525b2a2-1edc-11e8-9335-2a41126dee3c.png)
